projects/fsastorefrontlib/src/cms-components/checkout/components/configure-product/mini-cart/product-configuration-mini-cart.component.html

```diff
@@ -20,7 +20,7 @@ <h2 class="section-header-heading mb-0">
           >
             <div class="short-overview-title col-7">{{ label }}:</div>
             <div class="short-overview-value col-5">
-              {{ priceAttribute.value }}
+              {{ priceAttribute | currencyDetector | async }}
```

Comment:
Naming. 'Detector' part is sufficient here i think.
As i understood, this pipe is checking type of entry and if it's type is CURRENCY, it will transform value. 

Maybe to go with one of the following entryCurrency, fsCurrency, cartCurrency.

@SrckyD  what do you think?

Reply:
> Naming. 'Detector' part is sufficient...
You probably meant excess, not sufficient 😄 
And I agree, but since this is very specific and can be used only in mini-cart for the banking products, maybe the best naming would be `miniCartCurrency`